rell/src/proposal/poll.rell

```diff
@@ -5,12 +5,12 @@ entity poll {
 }
 
 entity poll_option {
-	key text;
+	key poll, text;
 }
 
 entity poll_vote {
-	key poll, account.user;
-	poll_option;
+	key poll, account.eth_account;
+	index poll, poll_option;
```

Comment:
is index nececcary when the poll is part of the primary key?

Reply:
If we want a query which shows everybody who voted for a certain poll option, we need index with structure (poll, poll_option). Now I'm not sure we need this query. Perhaps it would be relevant if we want to reward everyone who voted for a certain option.

Maybe we can just query all votes...